podcasts/AccountViewController.swift

```diff
@@ -9,6 +9,10 @@ class AccountViewController: UIViewController, ChangeEmailDelegate {
     static let newsletterCellId = "NewsletterCellId"
     static let actionCellId = "AccountActionCellId"
 
+    private var isUsernamePasswordLogin: Bool {
```

Comment:
I thought about that. There are a few reasons why I decided to go with this simple approach:

* I would need to persist that, which seems a little bit redundant because we already know if a login was done using username/password with what is currently persisted
* I would have to change the login flow to add this there. While this is not bad, it's more code
* I believe the near-term plan is to have at least a password reset so this code might die soon 💀 

I think we can wait until the need appears before doing something that may not be used?